patroni/ctl.py

```diff
@@ -1081,6 +1696,18 @@ def wait_until_pause_is_applied(dcs: AbstractDCS, paused: bool, old_cluster: Clu
 
 
 def toggle_pause(config: Dict[str, Any], cluster_name: str, group: Optional[int], paused: bool, wait: bool) -> None:
+    """Toggle the ``pause`` state in the cluster members.
+
+    :param config: Patroni configuration.
+    :param cluster_name: name of the Patroni cluster.
+    :param group: filter which Citus group we should toggle the pause state of. If ``None`` assume coordinator group.
+    :param paused: the desired state for ``pause`` in all nodes.
+    :param wait: ``True`` if it should block until the operation is finished or ``false`` for returning immediately.
+
+    :raises PatroniCtlException: if
+        * ``pause`` state is already *paused*; or
```

Comment:
```suggestion
        * ``pause`` state is already *paused*/*unpaused*; or
```

Reply:
```suggestion
        * Cluster ``is_paused`` state is already the same as *paused*; or
```

With this one I meant that `is_paused` attribute is already the same as `paused` argument.
The `*` is usually used to reference an argument, so I guess we should not use *unpaused*.
What do you think of the above instead?